src/Purestone/Player.hs

```diff
@@ -18,22 +18,26 @@ import Data.Aeson
 
 -------------------------------------------------------------------------------
 
--- | Reoresents the player during a match
+-- | Represents the player during a match
 data Player
-    = Player {
-        -- | The hand of the player
-        playerHand :: Hand,
-        -- | The deck of the player
-        playerDeck :: Deck,
-        -- | The current points the player has for their turn
-        playerCurrentPoints :: Int,
-        -- | The maximum number of points the player has
-        playerMaximumPoints :: Int,
-        -- | The Programs the player has played onto the board
-        playerBoardSpace :: [Card],
-        -- | The player's programs that have been killed
-        playerDeadPrograms :: [Card]
-    } deriving (Eq, Show, Generic)
+  = Player {
+    -- | The health of the player
+    playerHealth :: Int,
+    -- | The hand of the player
+    playerHand :: Hand,
+    -- | The deck of the player
+    playerDeck :: Deck,
+    -- | The current points the player has for their turn
+    playerCurrentPoints :: Int,
+    -- | The maximum number of points the player has
+    playerMaximumPoints :: Int,
+    -- | The Programs the player has played onto the board
+    playerBoardSpace :: [Card],
+    -- | The player's programs that have been killed
+    playerDeadPrograms :: [Card],
+    -- | How many over draws the player has had
+    playerOverDraw :: Int
+  } deriving (Eq, Show, Generic)
```

Comment:
I think this is the third time the indentation here is being changed now. We should stick with 4 spaces throughout the code, not 2.